src/services/codefixes/importFixes.ts

```diff
@@ -651,18 +668,28 @@ namespace ts.codefix {
     interface FixesInfo { readonly fixes: readonly ImportFix[]; readonly symbolName: string; }
     function getFixesInfo(context: CodeFixContextBase, errorCode: number, pos: number, useAutoImportProvider: boolean): FixesInfo | undefined {
         const symbolToken = getTokenAtPosition(context.sourceFile, pos);
-        const info = errorCode === Diagnostics._0_refers_to_a_UMD_global_but_the_current_file_is_a_module_Consider_adding_an_import_instead.code
-            ? getFixesInfoForUMDImport(context, symbolToken)
-            : isIdentifier(symbolToken) ? getFixesInfoForNonUMDImport(context, symbolToken, useAutoImportProvider) : undefined;
+        if (errorCode === Diagnostics._0_refers_to_a_UMD_global_but_the_current_file_is_a_module_Consider_adding_an_import_instead.code) {
+            return getFixesInfoForUMDImport(context, symbolToken);

```

Comment:
this now no longer calls `createPackageJsonImportFilter` and adds the `fixes` property. Is that OK? What did those do?

Reply:
Good catch, this was a mistake. (Our coverage for UMD import fixes is pretty sparse, which I think is a fair natural order of things.)